detail.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title></title>
+             <meta charset="utf-8" />
+        <link href="css/styles.css" type="text/css"
+              rel="stylesheet" />
+    </head>
+
+<body>
+
+<header>
+    <h1>H&amp;M</h1>
+    <a href="index.html">
+      <img src=images/logo.png />
+    </a>
+    <nav>
+        <ul>
+            <li><a id="current" href="#">Ladies</a></li>
+            <li><a href="#">Men</a></li>
+            <li><a href="#">Kids</a></li>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Sale</a></li>
+        </ul>
+    </nav>
+</header>
+
+<div class="row detail">
+    <p class="breadcrumb"><a href="#">Ladies &#62;</a><a href="#"> Everyday Dress &#62;</a> <a href="#"> Cardigans and Sweaters &#62; </a> <a href="index.html" id="current">Cardigans</a></p>
+    <h2>Knit Cardigan</h2>
+
+    <div class="detail-images">
+        <img class="detail-large" src="images/product-large.png" />
+        <img class="detail-small" src="images/detail1.png" />
+        <img class="detail-small" src="images/detail2.png" />
+        <img class="detail-small" src="images/detail3.png" />
+    </div>
+
+    <div class="product-description">
+        <p>Long cardigan in a soft, chunky knit with dropped shoulders, front pockets, and no buttons.</p>
+
+        <h3>Details</h3>
+        <ul class="detail-list">
+            <li>100&#37; acrylic. Machine-wash cold</li>
+            <li>Imported</li>
+            <li>Art.No. 86-2077</li>
+        </ul>
+
+    <div class="row order">
+
+        <form action="http://ww.example.com/order.php">
+            <p>
+                <label> Quantity&#58;</label>
+                <input id="quantity" type="text" name="quantity">
+            </p>
+            <p>
+                <input id="submit" type="submit" value="add to cart" />
+            </p>
+        </form>
+        </div>
+    </div>
+</div>
+
+
+<ul>
+    <li class="review">
+            <h3>Reviews</h3>
```

Comment:
This is a bit of a shortcut hack way to get this item aligned with the other reviews and it ends up bumping the content in this `.review` down making the spacing awkward. I've made a comment about setting the `min-height` but take that into consideration long with moving this `h3` out of the list and over the `ul` as a whole. This is a more semantic appropriate thing given that this heading really labels the whole list. Then you can wrap the `h3` and `ul` in a `div` and move the `width`, `margin`, etc. from `.review` items to the new grouping `div`. This will ensure the list and `h3` are aligned.